url0bbb975254_maxime_collin_mediatheque_tgz-pJ8-test_MainJ8/src/controleurs/ControleurUser.java

```diff
@@ -0,0 +1,174 @@
+package controleurs;
+
+import scolarite.Cours;
+import test.Bdd;
+import utilisateurs.Utilisateur;
+
+public class ControleurUser {
+	
+	ControleurEtudiant controleurEtudiant = new ControleurEtudiant();
+	ControleurEnseignant controleurEnseignant = new ControleurEnseignant();
+
+	public Boolean addUser(int idUser, String n, String p, String t, int nbEM) {
+		if (t == "Etudiant")
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**: An equality operator (== or !=) is used to compare strings, which matches strings based on address.

**Fix**: Do content comparison using the equals() method to compare the values of the strings.